manifest.toml

```diff
@@ -0,0 +1,44 @@
+name = "celestia-app"
+
+[defaults]
+builder = "exec:go"
+runner = "local:exec"
+
+[builders."docker:go"]
+enabled = true
+go_version = "1.17"
+build_base_image = "golang:1.17.8"
+enable_go_build_cache = true
+
+[builders."exec:go"]
+enabled = true
+go_version = "1.17"
+
+[runners."local:docker"]
+enabled = true
+
+[runners."local:exec"]
+enabled = true
+
+[runners."cluster:swarm"]
+enabled = true
+
+[runners."cluster:k8s"]
+enabled = true
+
+[[testcases]]
+name = "capp-1"
+instances = { min = 1, max = 100, default = 1 }
+
+    [testcases.params]
+    orc    = { type = "int", default = 1 }
+    app1   = { type = "string", default = "/core-configs/.celestia-app-1" }
+    app2   = { type = "string", default = "/core-configs/.celestia-app-2" }
+    app3   = { type = "string", default = "/core-configs/.celestia-app-3" }
+    app4   = { type = "string", default = "/core-configs/.celestia-app-4" }
+    app5   = { type = "string", default = "/core-configs/.celestia-app-5" }
+    app6   = { type = "string", default = "/core-configs/.celestia-app-6" }
+    app7   = { type = "string", default = "/core-configs/.celestia-app-7" }
+    app8   = { type = "string", default = "/core-configs/.celestia-app-8" }
+    app9   = { type = "string", default = "/core-configs/.celestia-app-9" }
+    app10   = { type = "string", default = "/core-configs/.celestia-app-10" }
```

Comment:
I've pre-generated them and uploaded already to this PR 

As well as we have a dedicated issue https://github.com/celestiaorg/test-infra/issues/24 for covering cases with gentx/collect-gentxs, etc.

Reply:
> I've pre-generated them and uploaded already to this PR

Can I reproduce generating the configs using this branch? I did not see any code that generates the configs (at least last time).

Reply:
I've uploaded the dummy version of the script as I want to proceed with prepping the app pkg. Take a look at `scripts/cf.sh`
Otherwise, this test-plan is already ready to be run, so you can just omit the key/gen/collect/config part and just run the network

I'll add the functionality later on #24 once i'm done with https://github.com/celestiaorg/celestia-node/issues/638

Reply:
Would be interesting if extracting some commonly used script makes sense: https://github.com/celestiaorg/go-cnc/pull/4#discussion_r885110082